experimental/datacommons-js/example-js.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.cdnfonts.com/css/museo-sans-rounded"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap/4.6.2/css/bootstrap.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://www.datacommons.org/css/ranking.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://www.datacommons.org/css/nl_interface.min.css"
+    />
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.21/lodash.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.7.0/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap/4.6.2/js/bootstrap.min.js"></script>
+    <script src="https://autopush.datacommons.org/datacommons.js"></script>
+    <style>
+      .my-chart {
+        height: auto;
+        margin-bottom: 2rem;
+      }
+    </style>
+    <script>
+      console.log("datacommons", datacommons);
```

Comment:
did you mean to keep this console log or was it just for testing?

Reply:
Removed